trans/views/translation.py

```diff
@@ -34,6 +35,9 @@ def get(self, request, *args, **kwargs):
         home_flat_page_slug = 'home-editor' if user.is_editor() else 'home'
         home_flat_page = FlatPage.objects.filter(slug=home_flat_page_slug).first()
         home_content = home_flat_page.content if home_flat_page else ''
+        translating_countries = User.objects.select_related('country').filter(~Q(groups__name__exact=['staff', 'editor']))
```

Comment:
Ah, just realized, these are `User` objects, can this be `translating_user` instead?